### PR TITLE
add new apphub example -- cifar10-fast and standardize all apphub example

### PR DESCRIPTION
**Pull request by geez0219**
1. Add new apphub example -- cifar10-fast
2. Standardize all existing apphub example in terms of get_estimator input
    Every get_estimator function has at least 4 arguments -- epoch, batch_size, max_steps_per_epoch, save_dir. It will benefit the future testing script to run custom test.

### CODE REVIEW

File: apphub/image_classification/cifar10_fast/cifar10_fast_tf.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+from fastestimator.pipeline import Pipeline
+from fastestimator.trace import Trace
+from fastestimator.trace.io import BestModelSaver
+from fastestimator.trace.metric import Accuracy
+
+policy = mixed_precision.Policy('mixed_float16')
+mixed_precision.set_policy(policy)
+
+
+class SmootOneHot(NumpyOp):
```
Comment (TortoiseHam) [inline apphub/image_classification/cifar10_fast/cifar10_fast_tf.py]: "SmoothOneHot", also, this should go into our op folder

Comment (TortoiseHam) [reply apphub/image_classification/cifar10_fast/cifar10_fast_tf.py]: rather, a parameterized version of this should go in our op folder

File: apphub/image_classification/cifar10_fast/cifar10_fast_tf.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            BestModelSaver(model=model, save_dir=save_dir, metric="accuracy", save_best_mode="max")
+        ],
+        max_steps_per_epoch=max_steps_per_epoch)
+
+    return estimator
+
+
+if __name__ == "__main__":
+    est = get_estimator()
+    est.fit()
```
Comment (TortoiseHam) [inline apphub/image_classification/cifar10_fast/cifar10_fast_tf.py]: @vbvg2008 has been putting .test() calls everywhere too. do we want that to be standard across all app hubs?

File: apphub/image_classification/cifar10_fast/cifar10_fast_torch.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    RandomCrop, Sometimes
+from fastestimator.op.tensorop.loss import CrossEntropy
+from fastestimator.op.tensorop.model import ModelOp, UpdateOp
+from fastestimator.pipeline import Pipeline
+from fastestimator.trace import Trace
+from fastestimator.trace.io import BestModelSaver
+from fastestimator.trace.metric import Accuracy
+
+
+class SmoothOneHot(NumpyOp):
```
Comment (TortoiseHam) [inline apphub/image_classification/cifar10_fast/cifar10_fast_torch.py]: like with the other one, this should move to central repo

File: apphub/image_classification/cifar10_fast/cifar10_fast_tf.py (hunk 2 above)
Comment (vbvg2008) [reply apphub/image_classification/cifar10_fast/cifar10_fast_tf.py]: in this case, yes.   but the test won't go to every case

File: apphub/image_classification/cifar10_fast/cifar10_fast_tf.py (hunk 1 above)
Comment (geez0219) [reply apphub/image_classification/cifar10_fast/cifar10_fast_tf.py]: done

File: apphub/image_classification/cifar10_fast/cifar10_fast_torch.py (hunk 3 above)
Comment (geez0219) [reply apphub/image_classification/cifar10_fast/cifar10_fast_torch.py]: done

**Pull request merged**